draft-ietf-spice-sd-cwt.md

```diff
@@ -66,11 +68,16 @@ The approach is based on Selective Disclosure JSON Web Token (SD-JWT), with chan
 
 This document updates the CBOR Web Token (CWT) specification {{RFC8392}}, enabling the holder of a CWT to disclose or redact special claims marked disclosable by the issuer of a CWT.
 The approach is modeled after SD-JWT {{-SD-JWT}}, with changes to align with conventions from CBOR Object Signing and Encryption (COSE) {{RFC9052}}.
-The ability to minimize disclosure of sensitive identity attributes, while demonstrating possession of key material and enabling a verifier to confirm the attributes have been unaltered by the issuer, is an important building block for many digital credential use cases.
-This specification brings selective disclosure capabilities to CWT, enabling application profiles to impose additional security criteria beyond the minimum security requirements this specification requires.
-Specific use cases are out of scope for this document.
-However, feedback has been gathered from a wide range of stakeholders, some of which is reflected in the examples provided in the appendix.
-
+This specification enables holders of CWT based credentials to prove the integrity and authenticity of selected attributes asserted by an issuer about a subject to a verifier.
+Although techniques such as one time use and batch issuance can improve the confidentiality and security characteristics of CWT based credential protocols, CWTs remain traceable.
+Selective Disclosure CBOR Web Tokens (SD-CWTs) are CWTs and can be deployed in protocols that are already using CWTs, even if they contain no optional do disclose claims.
+Credential types are distinguished by their attributes, for example a license to operate a vehicle and a license to import a product will contain different attributes.
+The specification of credential types is out of scope for this document, and the examples used in this document are informative.
+SD-CWT operates on CWT Claims Sets as described in {{RFC8392}}.
+CWT Claims Sets contain Claim Keys and Claim Values, which are expressed using the Major Types described in {{RFC8949}}.
```

Comment:
What is this sentence trying to accomplish? To anyone familiar with CBOR this is a extraneous. To anyone just trying to follow this conceptually this is going to be confusing.

Reply:
This "Claim" terminology is specific to CWT, not CBOR.

The intention was to address part of Mike's comment about map keys and cryptographic  keys being confusing, I'm repeating the CWT terminology here to try to address that, although I am more than happy to remove this text if we can avoid it.

Reply:
I was referring to ", which are expressed using the Major Types described in {{RFC8949}}." Can we please just nuke that?